students/grzegorz_bajorski/lesson_04_unit_testing/the_length_of_the_segment_tests.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from the_length_of_the_segment import distance
+
+
+class MyTest(pytest.TestCase):
+
+    def test_1(self):
+        self.assertRaises(TypeError, distance, None, 1, 2, 3)
```

Comment:
```self.assertRaises()``` is not pytest method. Here you should use:
```
with pytest.raises(TypeError):
        distance(None, 1,  2, 3)
```

Please have a look at second task description:
Test that it crashes with TypeError when called on None (use pytest.raises),